site/profile/manifests/core/icinga_resources.pp

```diff
@@ -0,0 +1,832 @@
+    }
+    | HTTP_TEMPLATE
+  $ping_svc_template = @("PING_TEMPLATE"/L)
+    {
+    "check_command": "hostalive",
+    "object_name": "${ping_svc_template_name}",
+    "object_type": "template",
+    "use_agent": true,
+    "zone": "master"
+    }
+    | PING_TEMPLATE
+  $dns_svc_template = @("DNS_TEMPLATE"/L)
+    {
+    "check_command": "dns",
+    "object_name": "${dns_svc_template_name}",
+    "object_type": "template",
+    "use_agent": true,
+    "zone": "master"
+    }
+    | DNS_TEMPLATE
+  $master_svc_template = @("MASTER_TEMPLATE"/L)
+    {
+    "check_command": "dhcp",
+    "object_name": "${master_svc_template_name}",
+    "object_type": "template",
+    "use_agent": true,
+    "zone": "master"
+    }
+    | MASTER_TEMPLATE
+  $ipa_svc_template = @("IPA_TEMPLATE"/L)
+    {
+    "check_command": "ldap",
+    "object_name": "${ipa_svc_template_name}",
+    "object_type": "template",
+    "use_agent": true,
+    "vars": {
+        "ldap_address": "localhost",
+        "ldap_base": "dc=lsst,dc=cloud"
+    },
+    "zone": "master"
+    }
+    | IPA_TEMPLATE
+  $disk_svc_template = @("DISK_TEMPLATE"/L)
+    {
+    "check_command": "disk",
+    "object_name": "${disk_svc_template_name}",
+    "object_type": "template",
+    "use_agent": true,
+    "vars": {
+        "disk_cfree": "10%",
+        "disk_wfree": "20%"
+    },
+    "zone": "master"
+    }
+    | DISK_TEMPLATE
+  ##Services Definition
+  #Ping, disk and RAM monitoring
+  $host_svc1 = @("HOST_SVC_1"/L)
+    {
+    "host": "${host_template}",
+    "imports": [
+        "${$ping_svc_template_name}"
+    ],
+    "object_name": "${host_svc_ping_name}",
+    "object_type": "object"
+    }
+    | HOST_SVC_1
+  $host_svc2 = @("HOST_SVC_2"/L)
+    {
+    "host": "${host_template}",
+    "imports": [
+        "${$disk_svc_template_name}"
+    ],
+    "object_name": "${host_svc_disk_name}",
+    "object_type": "object"
+    }
+    | HOST_SVC_2
+  #HTTP, Ping, disk and RAM monitoring
+  $http_svc1 = @("HTTP_SVC_1"/L)
+    {
+    "host": "${http_template}",
+    "imports": [
+      "${$http_svc_template_name}"
+    ],
+    "object_name": "${http_svc_name}",
+    "object_type": "object"
+    }
+    | HTTP_SVC_1
+  $http_svc2 = @("HTTP_SVC_2"/L)
+    {
+    "host": "${http_template}",
+    "imports": [
+        "${$ping_svc_template_name}"
+    ],
+    "object_name": "${http_svc_ping_name}",
+    "object_type": "object"
+    }
+    | HTTP_SVC_2
+  $http_svc3 = @("HTTP_SVC_3"/L)
+    {
+    "host": "${http_template}",
+    "imports": [
+        "${$disk_svc_template_name}"
+    ],
+    "object_name": "${http_svc_disk_name}",
+    "object_type": "object"
+    }
+    | HTTP_SVC_3
+  #DHCP, Ping, disk and RAM monitoring
+  $master_svc1 = @("MASTER_SVC_1"/L)
+    {
+    "host": "${master_template}",
+    "imports": [
+      "${$master_svc_template_name}"
+    ],
+    "object_name": "${master_svc_dhcp_name}",
+    "object_type": "object",
+    "vars": {
+      "dhcp_serverip": "139.229.135.5"
```

Comment:
Note that this is a site specific value.

Reply:
it should always match to foreman, so I'll base the logic in that